jobs/update-legal-filings/update_legal_filings.py

```diff
@@ -75,15 +79,17 @@ def check_for_manual_filings(application: Flask = None, token: dict = None):
     """Check for colin filings in oracle."""
     id_list = []
     colin_events = None
-    legal_url = application.config['LEGAL_URL']
+    legal_url = application.config['LEGAL_API_URL']
     colin_url = application.config['COLIN_URL']
     corp_types = [Business.TypeCodes.COOP.value, Business.TypeCodes.BC_COMP.value,
                   Business.TypeCodes.ULC_COMP.value, Business.TypeCodes.CCC_COMP.value]
     no_corp_num_prefix_in_colin = [Business.TypeCodes.BC_COMP.value, Business.TypeCodes.ULC_COMP.value,
                                    Business.TypeCodes.CCC_COMP.value]
```

Comment:
@argush3 this doesn't have BEN. Is it because a BEN cannot be modified in COLIN? do you remember?

Reply:
I am curious, do i need to add C types here

Reply:
> @argush3 this doesn't have BEN. Is it because a BEN cannot be modified in COLIN? do you remember?

yeah, i think BENs were a new business type that only started in the modernized system.  

I think we sync BENs to COLIN but COLIN doesn't actually filing anything for BENs I don't think.  So no need to COLIN -> LEAR for BENs.

Reply:
> I am curious, do i need to add C types here

yeah, probably need to add C types